charts/ocis/templates/nats/deployment.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   selector:
     matchLabels:
         app: nats
-  replicas: 1 #TODO: scale out ?
+  replicas: 1 #TODO: scale out? https://github.com/owncloud/ocis-charts/issues/12
```

Comment:
I know this is a tech preview, but IMHO Todos should not be in here.

Reply:
Unfortunately these more TODOs than this TODO, that are still to be done and to big / too much work for this PR. At least there is no more TODO without issue (link).

Would you prefer to have the TODO comments removed from the chart? I thought it's nice to have a direct link to the GitHub issue from the code.

Reply:
Sounds good, so at least we have a reference for tracking.